packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsUsersTooltip.tsx

```diff
@@ -69,32 +69,33 @@ export function CommentReactionsUsersTooltipContent(
 ) {
   const {currentUser, reactionName, userIds} = props
   const {t} = useTranslation(commentsLocaleNamespace)
-  const content = useMemo(() => {
+  const listFormat = useListFormat({style: 'long', type: 'conjunction'})
+
+  const UserList = useCallback(() => {
     const len = userIds.length
 
     if (len === 0 || !currentUser) return null
 
-    return userIds.map((id, index) => {
-      const separator = index < userIds.length - 1 && len > 2 && index !== userIds.length - 2
-      const showAnd = index === len - 1 && len > 1
-
-      return (
-        <Fragment key={id}>
-          {showAnd && (
+    return (
+      <>
+        {listFormat.formatToParts(userIds).map((item, index) =>
+          item.type === 'element' ? (
+            <InlineText weight="medium" key={item.value}>
```

Comment:
When there are two names displayed – there is no space between the first name and the "and" string (see image). I am not sure why this is happening or where a fix is best added, but the code below fixes the issue:

```ts
<>
  <InlineText weight="medium" key={item.value}>
    <UserDisplayName
      currentUserId={currentUser.id}
      isFirst={index === 0}
      userId={item.value}
    />
  </InlineText>

  {index === 0 && userIds.length === 2 && ' '}
</>
```


<img width="215" alt="Screenshot 2024-02-05 at 12 38 20" src="https://github.com/sanity-io/sanity/assets/15094168/9d9459da-1fd1-45f9-a043-125b85bb36b9">

Reply:
Is the problem only with 2 users? What happens with 3 users? Seems like the fix should be in the `listFormat`